DP3TApp/Logic/Tracing/TracingLocalPush.swift

```diff
@@ -146,6 +157,71 @@ class TracingLocalPush: NSObject {
         center.add(request1, withCompletionHandler: nil)
         center.add(request2, withCompletionHandler: nil)
     }
+
+    func handleTracingState(_ state: DP3TSDK.TrackingState) {
+        switch state {
+        case .active, .stopped:
+            resetAllErrorNotifications()
+        case let .inactive(error: error):
+            switch error {
+            case .bluetoothTurnedOff:
+                scheduleBluetoothNotification()
+            case let .exposureNotificationError(error: error):
+                if let error = error as? ENError {
+                    handleENError(error)
+                }
+            case .permissonError:
+                schedulePermissonErrorNotification()
+            default:
+                break
+            }
+        }
+    }
+
+    private func scheduleBluetoothNotification() {
+        scheduleErrorNotification(identifier: .bluetooth,
+                                  title: "bluetooth_turned_off_title".ub_localized,
+                                  text: "bluetooth_turned_off_text".ub_localized)
+    }
+
+    private func schedulePermissonErrorNotification() {
+        scheduleErrorNotification(identifier: .permission,
+                                  title: "tracing_permission_error_title_ios".ub_localized,
+                                  text: "tracing_permission_error_text_ios".ub_localized)
+    }
+
+    private func handleENError(_ error: ENError) {
+        switch error.code {
+        case .bluetoothOff:
+            scheduleBluetoothNotification()
+        case .notAuthorized, .notEnabled, .restricted:
+            schedulePermissonErrorNotification()
```

Comment:
This might never be called because iOS seems to kill EN apps when disabling them:

`Terminating ch.admin.bag.dp3t[2148] because access to the kTCCServiceExposureNotification service changed.`